function testRebaseWithError() {
    console.log('Error1');
    console.log('Error2');
    console.log('Error3');
}